receipt.php

```diff
@@ -69,13 +50,14 @@
     http_response_code(200);
     exit;
 } catch (Throwable $e) {
-    if (defined('WEBHOOK_ERROR_LOG_PATH')) {
-        $error_message = $e . "\n";
-        if (isset($notification)) {
-            $error_message = $notification->notificationId . ': ' . $error_message;
-        }
-        file_put_contents(WEBHOOK_ERROR_LOG_PATH, $error_message, FILE_APPEND);
+    $error_message = $e . "\n";
+    if (isset($notification)) {
+        $error_message = 
+            '[notificationId:' . $notification->notificationId . 
+            '; transactionId:' . $notification->payload->id . '] ' . $error_message;
```

Comment:
Do you perhaps want a space between the `[notificationId:x; transactionId:y]` prefix and the actual error content?